README.md

```diff
@@ -740,6 +740,48 @@ You should be able to pass data via `worker.workerData` (see [Custom Worker Opti
 
 Note that you cannot pass a built-in nor bound function.
 
+## Typescript Usage
+
+As is mentioned in this issue: https://github.com/breejs/bree/issues/24 and thanks to @ChrisEdgington.  You can use typescript workers as:
```

Comment:
And this link without ``.  This should did the trick

```md
As is mentioned in [#24](https://github.com/breejs/bree/issues/24) and thanks to @ChrisEdgington.
```

Reply:
Yes, much better